healthcare.management.system/rest-module/src/main/java/com/healthcare/management/system/rest/controller/UserController.java

```diff
@@ -0,0 +1,64 @@
+package com.healthcare.management.system.rest.controller;
+
+import com.healthcare.management.system.rest.entity.User;
+import com.healthcare.management.system.rest.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
```

Comment:
This is the root of the Controller api url path which is going to be shared across all methods for the User Resource. Therefore, you can set `@RequestMapping("/api/users")` at class level, then have e.g.:

@GetMapping("/")

@GetMapping("/{userId}")